operations/_scripts/generate/shorten_identifier.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+set -e
+
+IDENTIFIER="$1"
+final_id=""
+MAX_IDENTIFIER_LENGTH=$2
+
+
+if [ -z $MAX_IDENTIFIER_LENGTH ]; then
+  MAX_IDENTIFIER_LENGTH=60
+fi
+
+# if identifier is less than or equal to 60, shorten
+IDENTIFIER_LENGTH=${#IDENTIFIER}
+if (( $IDENTIFIER_LENGTH < $MAX_IDENTIFIER_LENGTH )) ; then
+  echo "$IDENTIFIER"
+  exit 0
+fi
+
+re='^[-_]*([[:alnum:]]*).*'
+if [[ $IDENTIFIER =~ $re ]]; then
+  while [[ $IDENTIFIER =~ $re ]]; do
+    if [ -z "${BASH_REMATCH[1]}" ]; then
+      break;
+    fi
+
+    # echo "BASE_REMATCH[0]"
+    # echo "${BASH_REMATCH[0]}"
+    # echo "BASE_REMATCH[1]"
+    # echo "${BASH_REMATCH[1]}"
+    # echo "BASE_REMATCH[2]"
+    # echo "${BASH_REMATCH[2]}"
+
+    # echo "==="
+    # echo "IDENTIFIER"
+    # echo "$IDENTIFIER"
```

Comment:
Please remove uneeded/unused debug code.

For someone looking at the code, this makes them think like the code is unfinished.